tailwind.config.ts

```diff
@@ -1,11 +1,14 @@
-import { tailwindConfig, withPenumbra } from '@penumbra-zone/ui/theme';
+import { withPenumbra, tailwindConfig } from '@penumbra-zone/ui/theme';
 
 export default withPenumbra({
   content: ['./src/**/*.{js,ts,jsx,tsx,mdx,css}'],
   theme: {
     extend: {
       colors: {
         ...tailwindConfig.theme.extend.colors,
+        app: {
+          main: '#0d0d0d',
```

Comment:
found that the background of pages in the new design is not a total black like in Minifront

Reply:
<img width="1728" alt="Screenshot 2024-11-19 at 9 01 24 PM" src="https://github.com/user-attachments/assets/a2c4ff38-0b67-492c-b4bf-763c5b3e0c1b">

Reply:
Thanks, fixed it!